templates/event-translations.php

```diff
@@ -0,0 +1,145 @@
+<?php
```

Comment:
We started using the `translations` template but it also includes header code. Also we need to do some tricks to get multiple of the tables working on the same page (e.g. the translations table has an `id="translations"` of which there cannot be multiple on a page).

It might be worth to propose to add the ability to have multiple translation tables on the same page in GlotPress but this is a) hard to argue, and b) not a quick PR. So I suggest to go this route for now.